test/types/mongodb.test-d.ts

```diff
@@ -11,3 +15,23 @@ expectDeprecated(Collection.prototype.remove);
 expectDeprecated(Collection.prototype.count);
 expectDeprecated(MongoError.create);
 expectDeprecated(AggregationCursor.prototype.geoNear);
+
+// test mapped cursor types
+const client = new MongoClient(process.env.MONGODB_URI || 'mongodb://localhost:27017');
```

Comment:
Not sure if this is the best approach, definitely open to suggestions about better ways of testing this.

Reply:
You can just pass in an empty string to satisfy typescript since the file is not actually executed